### PR TITLE
Add Víðarr design document and fix spelling in documentation

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: design.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Once a workflow is started (or restarted), it goes through phases:
+
+- the workflow waits for any consumable resource to allow it to continue
+- the workflow does a preflight check where it ensure that configuration in the output metadata is valid
+- it provisions in any input files
+- it runs the workflow
+- it provisions out the results of the workflow
+- it performs any clean up required
+
+We keep running into issue of running out of temp. Shesmu can throttle, but
```
Comment (callunity) [inline design.md:92]: temp = temp space?

File: design.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+it's often too late. The Víðarr has a number of consumable resources (_e.g._,
```
Comment (callunity) [inline design.md:93]: The Vidarr -> Vidarr? Or the Vidarr server?

Comment (apmasell) [reply design.md:93]: I think Vidarr server makes more sense.

**Review by callunity — APPROVED** [review]

File: admin-guide.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ For workflow developers, it is also useful to install a `/srv/vidarr/cli` script
     #!/bin/sh
     exec /usr/lib/jvm/java-14-openjdk-amd64/bin/java -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:7071 --module-path $(ls /srv/vidarr/jars/*.jar|tr '\n' :) -m ca.on.oicr.gsi.vidarr.cli/ca.on.oicr.gsi.vidarr.cli.Main "$@"
 
-This file does _not_ have to be on the Vidarr server and a copy of JARs and this script can be placed on another server. They will also need a configuration file to be able to run workflows in testing. It is a reduced from of the main configuration file:
+This file does _not_ have to be on the Víðarr server and a copy of JARs and this script can be placed on another server. They will also need a configuration file to be able to run workflows in testing. It is a reduced from of the main configuration file:
```
Comment (asofmosq) [inline admin-guide.md:155]: It is a reduced from of -> reduced form of

File: design.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+have outsourced the workflow engine with the Broad's
+[Cromwell](https://cromwell.readthedocs.io/en/stable/). Víðarr is our
+replacement for the metadata tracking.
+
+It schedules workflows using a workflow engine and collects the
+output from these workflows and stores the metadata about these files and
+connections to the Pinery LIMS information. It supports using Cromwell
+as a workflow engine, with the future goal to support other workflow engines.
+It has to support provisioning out other kinds of data and associate LIMS
+metdata, with the future goal to provision out logs, QC information, and
```
Comment (asofmosq) [inline design.md:16]: metadata

**Review by asofmosq — APPROVED** [review]

**Pull request merged**